### PR TITLE
fix: restructure vc-verification package

### PR DESCRIPTION
**Pull request by nichonien**

### CODE REVIEW

File: packages/vc-verification/src/utils/formatters.ts [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+ * @param {string} did
+ * @returns {string} DID address in format "did:" method-name ":" method-specific-id ":" address
+ */
+function upgradeDidWithChainId(did: string): string {
+  const { foundChainInfo } = getDIDChain(did);
+  if (foundChainInfo) return did;
+
+  const didParts = did.split(':');
+
+  const chainName = 'volta';
```
Comment (JGiter) [inline packages/vc-verification/src/utils/formatters.ts:43]:
```suggestion
  const chainName = Chain.Volta;
```

File: packages/vc-verification/src/utils/formatters.ts [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+        roleJwt.payload[key] = upgradeDidWithChainId(expectedEthrDID);
+      } else {
+        invalidDIDProperty = true;
+      }
+    }
+  }
+  return invalidDIDProperty ? undefined : roleJwt;
+}
+
+/**
```
Comment (JGiter) [inline packages/vc-verification/src/utils/formatters.ts:33]:
```suggestion
/**
* Adds chain id to DID
```

**Review by JGiter — APPROVED** [review]

**Review by JGiter — DISMISSED** [review]
Looks good, but please don't include `fix: minor changes` in history

**Review by jrhender — APPROVED** [review]
Seems fine to me

**Pull request merged**